gdnative-core/src/core_types/color.rs

```diff
@@ -23,6 +24,22 @@ impl Color {
         Color { r, g, b, a: 1.0 }
     }
 
+    #[inline]
+    pub fn hsv(h: f32, s: f32, v: f32) -> Color {
+        Color::hsva(h, s, v, 1.0)
+    }
+
+    #[inline]
+    pub fn hsva(h: f32, s: f32, v: f32, a: f32) -> Color {
```

Comment:
Should be `from_hsva` as we agreed in https://github.com/godot-rust/godot-rust/pull/729#discussion_r622177710?

Reply:
This should be fixed as of 17e84d5.